### PR TITLE
Standardize users subdirectory

### PR DESCRIPTION
**Pull request by dav3r**
# 

## 🗣 Description

@jsf9k and I discussed making some changes to the roles, policies, variables, etc. that will appear in every account subdirectory. These are the changes appropriate to the "users" subdirectory.

## 💭 Motivation and Context

Ordo ab chao

## 🧪 Testing

Testing was done by executing a `terraform plan` and verifying that no syntax errors were reported.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: users/README.md [hunk 1]
```diff
@@ -73,17 +73,21 @@ future changes by simply running `terraform apply
...
 | assume_access_terraform_backend_policy_name | The name to assign the IAM policy that allows assumption of the role with access to the Terraform backend | string | `Terraform-AssumeAccessTerraformBackend` | no |
 | assume_iam_admin_policy_description | The description to associate with the IAM policy that allows assumption of the role to become an IAM admininistrator | string | `Allow assumption of the IamAdministrator role.` | no |
 | assume_iam_admin_policy_name | The name to assign the IAM policy that allows assumption of the role to become an IAM admininistrator | string | `AssumeIamAdministrator` | no |
-| aws_region | The AWS region where the non-global resources for this account are to be created (e.g. us-east-1) | string | `us-east-1` | no |
+| assume_tf_provisionaccount_policy_description | The description to associate with the IAM policy that allows assumption of the role with sufficient permissions to provision all AWS resources in the Terraform account | string | `Allow assumption of the ProvisionAccount role in the Terraform account.` | no |
```
Comment (felddy) [inline users/README.md]: What is a Provis Ion Account?

File: users/README.md [hunk 2]
```diff
@@ -73,17 +73,21 @@ future changes by simply running `terraform apply
...
 | iam_admin_role_name | The name to assign the IAM role that allows full IAM administrator access in this account | string | `IamAdministrator` | no |
-| terraform_backend_users_group | The name of the group to be created for users allowed to access the terraform backend | string | `terraform_backend_users` | no |
+| iam_admins_group | The name of the group to be created for users allowed to be IAM administrators | string | `iam_admins` | no |
 | tags | Tags to apply to all AWS resources created | map(string) | `{}` | no |
+| terraform_account_id | The ID of the Terraform account, which contains roles that can be assumed to access the Terraform backend and to provision AWS resouces in that account. | string | | yes |
+| terraform_account_provisioners_group_name | The name of the group to be created for users allowed to provision the Terraform account | string | `terraform_account_provisioners` | no |
+| terraform_backend_users_group | The name of the group to be created for users allowed to access the Terraform backend | string | `terraform_backend_users` | no |
```
Comment (jsf9k) [inline users/README.md]: Should this be `terraform_backend_users_group_name`, to be consistent with the previous variable?

Comment (felddy) [reply users/README.md]: or `terraform_backendusersgroup_name` 😝

Comment (dav3r) [reply users/README.md]: You are correct sir - fixed in https://github.com/cisagov/cool-accounts/pull/4/commits/9e1d74064d54dcd1016ea3319dff7a3f119a15a5

File: users/README.md (hunk 1 above)
Comment (dav3r) [reply users/README.md]: thankyou

**Pull request merged**